src/OpenTelemetry.Instrumentation.SqlClient/README.md

```diff
@@ -221,6 +221,37 @@ using var tracerProvider = Sdk.CreateTracerProviderBuilder()
     .Build();
 ```
 
+## Filter
+
+This option allows to filter out activities based on the properties of the
+`SqlCommand` object being instrumented using a `Func<object, bool>`.
```

Comment:
https://github.com/open-telemetry/opentelemetry-dotnet/tree/main/src/OpenTelemetry.Instrumentation.AspNetCore#filter the wording here is slightly better. Not blocking. There are plans to change the return type from bool to more easy-to-use Enums like DropActivity, ReportActivity etc, soon.